{{ cookiecutter.repo_name }}/buildSrc/src/main/java/plugins/AppCenterConfigPlugin.kt

```diff
@@ -0,0 +1,49 @@
+package plugins
+
+import com.betomorrow.gradle.appcenter.extensions.AppCenterExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import utils.GitHelper
+import utils.propOrEmpty
+
+class AppCenterConfigPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        if (System.getenv("CI") == "true") {
+            target.plugins.apply("com.betomorrow.appcenter")
+            val extension = target.extensions.getByType(AppCenterExtension::class.java)
+            try {
+                extension.configure(target)
+            } catch (e: Exception) {
+                e.printStackTrace()
+            }
+        }
+    }
+
+    private fun AppCenterExtension.configure(target: Project) {
+        val gitHelper = GitHelper(target.rootDir)
+
+        apiToken = target.propOrEmpty("APPCENTER_TOKEN")
+        ownerName = "RightpointApps"
+        notifyTesters = false
+        apps {
```

Comment:
We might be able to do something more generic here maybe.

Reply:
Interesting idea, I can look into this.

Reply:
It's just a suggestion, not sure this is doable, but it might worst the try.

Reply:
To follow up here; this is probably doable but the one thing I wasn't sure how to handle were the upload's release notes. We currently tie the release notes to specific build variants and these different build variants take advantage of very specific git commands. Again, probably possible to do, but at the moment I'm not entirely sure how to handle this in a generic way.

Reply:
It something we can improve later.